packages/noya-renderer/src/components/SketchFileRenderer.tsx

```diff
@@ -169,6 +170,49 @@ export default memo(function SketchFileRenderer() {
     [page, state.selectedObjects],
   );
 
+  const distanceLabelAndPathBetweenSketchLayers = useMemo(() => {
+    if (
+      !highlightedLayer ||
+      !highlightedLayer.isMeasured ||
+      !boundingRect ||
+      state.selectedObjects.length === 0 ||
+      state.selectedObjects.includes(highlightedLayer.id)
+    ) {
+      return;
+    }
+
+    const indexPath = findIndexPath(
+      page,
+      (layer) => layer.do_objectID === highlightedLayer.id,
+    );
+
+    if (!indexPath) return;
+
+    const highlightedBoundingRec = getBoundingRect(
+      page,
+      AffineTransform.identity,
+      [highlightedLayer.id],
+      {
+        clickThroughGroups: true,
+        includeHiddenLayers: true,
+      },
+    );
+
+    const selectedAndHighlightedLayers = {
```

Comment:
Can we remove this object and instead pass the props directly?

Very minor: I would probably name variables with the suffix `Rect` rather than `Rec`, since the type is `Rect`. And the props `selectedLayer` and `highlightedLayer` could probably also be `selectedRect` and `highlightedRect`, for consistency (`Layer` and `Rect` are different types)

Reply:
changed naming and removed object to pass props directly